chapters/5-processing.tex

```diff
@@ -828,18 +828,28 @@ \subsection{Documenting data cleaning}
 so other researchers may use them for their analysis as well.
 
 \section{Looking ahead}
-Once you have corrected, cleaned, 
-and de-identified one or several data tables,
-and compiled the documentation to support it,
+This chapter introduced a process of exploring, understanding, and correcting
+the data that you obtained from the field or from partners.
+This process, called data cleaning and processing,
+creates your first research outputs using original data:
+an analysis-ready dataset.
+This dataset is a clear representation of reality (it is ``tidy''),
+it faithfully represents the measurements it was intended to collect,
+and it does not expose the identities of the people described by it.
+You have also now taken the time to fully understand the patterns and structures
+in your data, and annotated and labeled them for your use and use by others.
+Once you have completed these steps and
+and compiled the documentation to accompany your work,
 you have created the first research output of your project: an original dataset.
-This will help you organize your files and create a backup of the data,
-and some donors require that the data be filed as an intermediate step of the project.
-However, the reason why you acquired and cleaned this data
-was a research question.
+Combined with the data map, this product 
+will be the fundamental starting point for all analysis work.
+Archiving, cataloging, and publishing this dataset 
+will organize your files and create a backup of the data.
 Though data cleaning is typically the most time-consuming process in the data work,
-the reason why it is done is preparing data to be analyzed.
-In Chapter 6, you will see what are the last few steps needed to finally run the
-regressions originally specified in your analysis plan
+the reason why it is done is preparing data to be analyzed
+to answer a set of research questions.
```

Comment:
this sentence is also a bit awkward. do we need it? not sure of the point it's trying to make. maybe something like, although the process described in this chapter is typically very time-consuming, investing in a careful research product at this point will simplify all workflows going forward

Reply:
b7bbbdb330e6b9c6c35df47ec4d84e7a88992833